### ISSUE TITLE
Link opens in browser after Render with qml file when hypothes.is comments are turned on in _quarto.yml

### ISSUE THREAD

**Issue by petrbouchal** [rstudio/rstudio#12413]
### System details

    RStudio Edition : Desktop
    RStudio Version : 2023.03.0-daily+3 (2023.03.0-daily+3)
    OS Version      :  MacOS 13.1 Beta (22C5050e)
    R Version       : 4.2.1 (2022-06-23)

### Steps to reproduce the problem

Create a Quarto file - e.g. index.qmd as below

````
---
title: "Test"
format: html
---

````

Create a _quarto.yml file as below:

````
project:
  output-dir: docs
  type: website
website:
  comments:
    hypothesis: true
````

Click Render

### Describe the problem in detail

When the preview opens in the Preview pane, the following link opens (twice) in the default browser:

https://hypothes.is/app.html#config=%7B%22openSidebar%22%3Afalse%2C%22showHighlights%22%3A%22always%22%2C%22origin%22%3A%22https%3A%2F%2Fhypothes.is%22%2C%22version%22%3A%221.1165.0%22%2C%22hostURL%22%3A%22http%3A%2F%2Flocalhost%3A5251%2Findex.html%3Fcapabilities%3D1%26host%3Dhttp%253A%252F%252F127.0.0.1%253A20523%22%7D

### Describe the behavior you expected

I expect the HTML file to be shown in the Preview pane and nothing to open in the browser.

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [ ] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by ronblum** [comment]
@petrbouchal Thanks for letting us know! You can probably guess my first question: Did this work in previous versions of RStudio?

Also, thank you for trying out the Cherry Blossom so early! I appreciate your trying out such an early development release.

**Comment by petrbouchal** [comment]
Yes, this issue also manifests on the current stable version, but unfortunately I don't recall when it first appeared.

**Comment by petrbouchal** [comment]
Just noticed that when a rendered Quarto HTML output is in the Preview pane, this behaviour (hypothes.is link being opened in browser) is also triggered by opening a new source pane (Cmd+7).

**Comment by ronblum** [comment]
@petrbouchal I can reproduce it (that was easy) in 2022.12.0+353, and it works fine in 2022.07.2+576. This might be a duplicate of #12494—it looks and feels the same—but I'll leave this open for the moment.

**Comment by ronblum** [comment]
Verified fixed using RStudio Desktop 2023.03.0-daily+323 on MacOS 13.2.

**Comment by rmflight** [comment]
I hadn't thought to file an issue when I was encountering this behavior with my giscus comment enabled quarto blog.

But I've also confirmed that 2023.03.0-daily+328 no longer has this behavior. Thanks so much!
